src/librustc_mir/repr.rs

```diff
@@ -310,18 +310,13 @@ impl<'tcx> Debug for Terminator<'tcx> {
     fn fmt(&self, fmt: &mut Formatter) -> Result<(), Error> {
         use self::Terminator::*;
         match *self {
-            Goto { target } =>
-                write!(fmt, "goto -> {:?}", target),
-            Panic { target } =>
-                write!(fmt, "panic -> {:?}", target),
-            If { cond: ref lv, ref targets } =>
-                write!(fmt, "if({:?}) -> {:?}", lv, targets),
+            Goto { target } => write!(fmt, "goto -> {:?}", target),
+            Panic { target } => write!(fmt, "panic -> {:?}", target),
+            If { cond: ref lv, ref targets } => write!(fmt, "if({:?}) -> {:?}", lv, targets),
             Switch { discr: ref lv, ref targets } =>
                 write!(fmt, "switch({:?}) -> {:?}", lv, targets),
-            Diverge =>
-                write!(fmt, "diverge"),
-            Return =>
-                write!(fmt, "return"),
+            Diverge => write!(fmt, "diverge"),
+            Return => write!(fmt, "return"),
```

Comment:
Hmm, good question. We probably should care a bit about consistency, but we don't at the moment (non-local reasoning is very rare in rustfmt atm).